core/src/main/java/org/neo4j/ogm/autoindex/AutoIndexManager.java

```diff
@@ -105,7 +79,7 @@ public AutoIndexManager(MetaData metaData, Configuration configuration, Neo4jSes
      * Builds indexes according to the configured mode.
      */
     public void build() {
```

Comment:
Rename to describe what happens here. It does not necessarily build an index.
(I know that this is an additional change and not part of the PR)

Reply:
Added a new (delegate) method and deprecated the old one.